couchbase/kv/v1/kv.proto

```diff
@@ -41,6 +41,11 @@ enum DurabilityLevel {
   DURABILITY_LEVEL_PERSIST_TO_MAJORITY = 2;
 }
 
+enum CompressionEnabled {
```

Comment:
What sort of compression is this requesting - Snappy?  Do we want to embed that here to allow other compression schemes in future?

Reply:
It's always snappy, if we added additional compression schemes in the future, we would likely just expand this enum and the one-of.